route_test.go

```diff
@@ -21,6 +22,64 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestRecoverWithDefaultCallback(t *testing.T) {
+	globalRecoverCallback = nil
+	mockConfig := configmocks.NewConfig(t)
+	mockConfig.On("GetBool", "http.drivers.fiber.prefork", false).Return(false).Once()
+	mockConfig.On("GetInt", "http.drivers.fiber.body_limit", 4096).Return(4096).Once()
+	mockConfig.On("GetInt", "http.drivers.fiber.header_limit", 4096).Return(4096).Once()
+
+	route, err := NewRoute(mockConfig, nil)
+	assert.Nil(t, err)
+	route.Recover(globalRecoverCallback)
```

Comment:
This line is unnecessary in this case, we don't need to pass `nil` into the `Recover` method.

Reply:
in goravel/gin we used gin.Recovery() and the default recovery test covered this case. Is it necessary to test the default recoverer in goravel/fiber?

Reply:
we can initially set globalRecoverGallback
```globalRecoverCallback := func(ctx contractshttp.Context, err any) {
		ctx.Request().AbortWithStatusJson(http.StatusInternalServerError, fiber.Map{"error": "Internal Server Error"})
	}```

Reply:
There is default recovery as well:

```go
fiberHandlers := []fiber.Handler{
  fiberrecover.New(fiberrecover.Config{
	  EnableStackTrace: debug,
  }),
}
```

Setting a default `globalRecoverCallback`is a good idea.